07-React/react-boilerplate-master/views/moviecrud.js

```diff
@@ -0,0 +1,59 @@
+var React = require ('react');
```

Comment:
I'm not sure if using *CRUD* as a name identifier would be good (maybe functionality requirements could change in the future...). Something more generic could work better.